example_test.go

```diff
@@ -651,3 +651,43 @@ func ExampleConnection_Execute() {
 	fmt.Println("MetaData", resp.MetaData)
 	fmt.Println("SQL Info", resp.SQLInfo)
 }
+
+// To use prepared statements to query a tarantool instance, call Prepare.
+func ExampleConnection_Do() {
```

Comment:
```suggestion
func ExamplePreparedStatement() {
```
see https://go.dev/blog/examples .

Reply:
> func ExampleBar_Qux() // documents the Qux method of type Bar

Reply:
Yeah, I didn't think it was related to `Connection.Do()`. It may be better to connect it with the type `PreparedStatement` (my suggestion) or `Connection.NewPreparedStatement()` (your update). I'm ok now.